packages/rewards-calculator/src/workers.ts

```diff
@@ -76,7 +89,7 @@ export class Workers {
 
   public async getStakes() {
     const stakes = await getStakes(this, this.nextDistributionStartBlockNumber);
-    this.parseMulticallResult("stake", stakes);
+    this.parseMulticallResult("stake", stakes, bigIntToDecimal);
```

Comment:
maybe add something like `mapMulticallResult(result, mapping)` and call like
`this.parseMulticallResult("stake", mapMulticallResult(stakes, bigIntToDecimal));`